recipes/mysql-concpp/all/conanfile.py

```diff
@@ -0,0 +1,240 @@
+import os
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import is_apple_os
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.files import get, copy
+from conan.tools.build import check_min_cppstd, cross_building
+from conan.tools.files import get, replace_in_file
+from conan.tools.scm import Version
+from conan.tools.microsoft import is_msvc, is_msvc_static_runtime
+
+required_conan_version = ">=1.64.1"
+
+class MysqlCppConnRecipe(ConanFile):
+    name = "mysql-concpp"
+    package_type = "library"
+    short_paths = True
+    version= "9.0.0"
+
+    # Optional metadata
+    license = "GPL-2.0"
```

Comment:
```suggestion
    license = "GPL-2.0-only"
```
Can be made more specific.